miniwin/src/d3drm/backends/software/renderer.cpp

```diff
@@ -62,6 +63,15 @@ void Direct3DRMSoftwareRenderer::ClearZBuffer()
 		_mm_empty();
 	}
 #endif
+#elif defined(__3DS__)
+	SDL_Log("FIXME!");
```

Comment:
what's the specific issue here?

Reply:
Compiler error.
```cpp
/opt/devkitpro/devkitARM/lib/gcc/arm-none-eabi/13.2.0/include/arm_neon.h:6784:1: error: inlining failed in call to 'always_inline' 'float32x4_t vdupq_n_f32(float32_t)': target specific option mismatch
 6784 | vdupq_n_f32 (float32_t __a)
      | ^~~~~~~~~~~
/Users/jpeisach/isle-3ds/miniwin/src/d3drm/backends/software/renderer.cpp:69:47: note: called from here
   69 |                 float32x4_t inf4 = vdupq_n_f32(inf);
      |                                    ~~~~~~~~~~~^~~~~
      ```
      (cut down)

Reply:
Looks like 3DS doesn't support neon so just wrap this portion in `!defined(__3DS__)`